src/base/rss/rss_feed.cpp

```diff
@@ -424,6 +410,86 @@ void Feed::downloadIcon()
             , this, &Feed::handleIconDownloadFinished);
 }
 
+int Feed::updateArticles(QList<QVariantHash> loadedArticles)
+{
+    if (loadedArticles.count() == 0)
+        return 0;
+
+    QDateTime dummyPubDate {QDateTime::currentDateTime()};
+    QVector<QVariantHash> newArticles;
+    newArticles.reserve(loadedArticles.size());
+    for (QVariantHash article : loadedArticles) {
+        QVariant &torrentURL = article[Article::KeyTorrentURL];
+        if (torrentURL.toString().isEmpty())
+            torrentURL = article[Article::KeyLink];
+
+        // if item does not have a guid, fall back to some other identifier
```

Comment:
@glassez:  **Grammar Alert:**

>**If** item does not have a **GUID**, fall back to some other **identifier.** 

I understand that this line was simply moved, but better capitalization should still improve its readability.

Reply:
Thank you! Fixed.